ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml

```diff
@@ -3,7 +3,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-9-release-golang-1.20-openshift-4.14
+    tag: rhel-9-release-golang-1.20-openshift-4.15
```

Comment:
This should be 4.16, shouldn't it?

Reply:
Installer uses 4.15... but let's see if 4.16 is available already.